guides/source/active_job_basics.md

```diff
@@ -70,12 +70,14 @@ Here's what a job looks like:
 class GuestsCleanupJob < ActiveJob::Base
   queue_as :default
 
-  def perform(*args)
+  def perform(*guests)
     # Do something later
   end
 end
 ```
 
+Note that you can define `perform` with as many arguments you want.
```

Comment:
There's just a missing "as" before you.

Could you also squash your commits together please ? Then it should be good to go! :-)